src/nomad_simulations/schema_packages/basis_set.py

```diff
@@ -184,22 +184,444 @@ def normalize(self, archive: 'EntryArchive', logger: 'BoundLogger') -> None:
+        for the s-part and the p-part.
+        """,
+    )
+
+    exponents = Quantity(
+        type=np.float32,
+        shape=['n_primitive'],
+        description="""
+        The **exponents** of each primitive basis function. 
+        In a Gaussian basis set (GTO), these are the alpha_i in 
+        exp(-alpha_i * r^2). In a Slater-type basis (STO), they'd be 
+        exp(-zeta_i * r). Typically sorted from largest to smallest.
+        """,
+    )
+
+    primitive_normalization = Quantity(
+        type=MEnum('L2', 'none', 'contracted', 'custom'),
+        default='L2',
+        description="""
+        Convention used for contraction coefficients:
+        • 'L2' - coefficients multiply L²-normalised primitives (Gaussian/BSE style).  
+        • 'none' - coefficients multiply unnormalised primitives (rare).  
+        • 'contracted' - primitives normalised, contracted function renormalised (GAMESS-style).  
+        • 'custom' - explicit factors are stored in `primitive_factor` / `shell_factor`.
+        """,
+    )
+
+    primitive_factor = Quantity(
+        type=np.float64,
+        shape=['n_primitive'],
+        description="""
+        **Extra scaling factors for each primitive** (dimensionless).
+
+        - Let ϕₖ(r) be the *analytic* primitive basis function (Gaussian or
+          Slater) with its standard L² normalisation constant **N_k**.
+        - Many libraries store coefficients **cₖ** that *already* include N_k
+          (Gaussian convention).  Others store raw cₖ and rely on the program
+          to multiply by N_k at run time.
+        - `primitive_factor[k]` captures *any* additional multiplier applied
+          to ϕₖ beyond its analytic normalisation—e.g. if the published
+          coefficients correspond to **unnormalised** primitives,
+          `primitive_factor[:] = 1 / N_k`.
+
+        Typical values:
+
+        | Scenario (source)                               | primitive_factor |
+        |-------------------------------------------------|------------------|
+        | BSE / Gaussian input (“coeffs x L²-norm prim”)  | 1.0              |
+        | Raw CRYSTAL deck (“coeffs for unnorm prim”)     | 1 / N_k          |
+        | Custom numeric STOs with internal scaling       | arbitrary        |
+
+        If omitted or a value of 1.0 is supplied, the primitives follow the
+        convention implied by `primitive_normalization`.  Only set a custom
+        factor when `primitive_normalization == 'custom'`.
+        """,
+    )
+
+    shell_normalization = Quantity(
+        type=np.float64,
+        description="""
+        **Global normalisation constant for the contracted shell** χ(r).
+
+        After primitives (and their `primitive_factor`s) are combined with the
+        contraction coefficients, one may optionally rescale the entire
+        contracted function so that
+
+        \\[
+            \\int_{\\mathbb{R}^3} |χ(r)|^2 \\,dr = 1.
```

Comment:
same here, does this display? I remember looking into putting equations in the descriptions but I can't remember the outcome now. 

I'm not suggesting you change anything, but if it doesn't display in the GUI we could talk to Area D about a feature for this